lessons/table-and-display-functions/langs/en-us/pages/exploring-displays-alt.adoc

```diff
@@ -0,0 +1,102 @@
+= Exploring Displays
```

Comment:
I would swap the columns in each subtable, so that the _first_ thing kids do is draw a pretty picture.